src/lightning_lite/CHANGELOG.md

```diff
@@ -51,6 +51,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Restored sampling parity between PyTorch and Lite dataloaders when using the `DistributedSampler` ([#16101](https://github.com/Lightning-AI/lightning/issues/16101))
 
 
+## [1.8.6] - 2022-12-21
+
+- minor cleaning
+
+
```

Comment:
What does that mean? this is not very informative :))

Reply:
yes, there was no change but was feeling bad to keep it completely empty :D

Reply:
Whats the issue with saying "No changes"?

Reply:
why would you release something with no changes...

Reply:
i think there was some CI related cleaning